hilti/toolchain/include/compiler/detail/cxx/elements.h

```diff
@@ -48,6 +48,7 @@ class Element {
     bool isMultiLine() const { return _s.find('\n') != std::string::npos; }
 
     operator std::string() const { return _s; }
+    operator std::string_view() const { return _s; }
```

Comment:
With this overload present ideally we would never call the the implicit conversion to `std::string` anymore since it always creates a copy. Currently we cannot simply remove it since many functions expect e.g., `const std::string&` args, so I wonder whether it wouldn't make sense to get rid of both these implicit conversions and instead provide an explicit function

```cpp
    const std::string& str() const { return _s; }
```

Reply:
And/or: make more methods take views. But either way that looks like a larger change not really that related to this PR, so I'll leave it for now.